### ISSUE TITLE
Traning and testing losses return 'nan' during training

### ISSUE THREAD

**Issue by DataExMachina** [kdgutier/esrnn_torch#5]
Hi all,

First of all thank you for your amazing job on ES-RNN pytorch implementation.
I have been able to run the M4 scripts, but I encountered troubles trying to run ESRNN architecture on an another data source. 

I am trying to apply this model on the M5 competition. The input is the following tables:
![image](https://user-images.githubusercontent.com/25567465/80121647-dee9c780-858c-11ea-938e-baa3ecd33a6f.png)

I define the model as following (taking some parameters from the initial config dict) and use previously defined tables to train it:
![image](https://user-images.githubusercontent.com/25567465/80121910-22dccc80-858d-11ea-8d71-dba4d5bd9303.png)

But I observe that the network does not seem to learn, because of 'nan' returned during training. Is there something I missed ?

**Comment by cristianchallu** [comment]
Hi,

Thanks for using our implementation. For what I can see in the data there are multiple zeros in the target variable. The model will return nan losses if the target variable of any serie has zero in the first ds (since the model normalizes the series by previous values). Can you try adding 1 (or any scalar) to all the target variable for all ds? You can then substract the scalar to the predictions.

**Comment by DataExMachina** [comment]
Hi !

Thanks a lot, it works !

**Comment by FangSimon** [comment]
Hi there, 

I am running into a similar problem. While I do get a value for training loss and testing loss, I get a nan for my evaluation metrics: sMAPE, MASE and OWA, when I am fitting the model. 

> Infered frequency: W-SUN
> =============== Training ESRNN  ===============
> 
> ========= Epoch 0 finished =========
> Training time: 12.04543
> Training loss (50 prc): 0.49441
> Testing loss  (50 prc): 0.02670
> OWA: nan 
> SMAPE: nan 
> MASE: nan

I have checked and there are no zeroes in the first ds. What could be causing the problem?
